src/components/DownloadComponent.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div>
+    <h2 class="mt-5">Download af PDF</h2>
+    <p class="mb-5">
+      Eksempel på at downloade noget HTML som en PDF. Når applikationen afvikles uden for Virksomhedsguiden, vil kaldet altid fejle, da det ikke er
```

Comment:
Måske lave en info kasse, som kun vises når den afvikles uden for Virksomhedsguiden der advarer at man kalder en PROD service og derfor ikke må misbruges.

Reply:
Viser nu en info kasse om at download fejler pga CORS, hvis man kører applikationen selvstændigt